src/labels/CustomLabels.labels

```diff
@@ -6237,6 +6237,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message description for when the tokenization request times out</shortDescription>
         <value>The request timed out. Please try again or contact your Salesforce administrator.</value>
     </labels>
+    <labels>
+        <fullName>gePaymentServicesConfigError</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Payment Services configuration error</shortDescription>
+        <value>Elevate customer configuration not found. Please complete Elevate onboarding.</value>
```

Comment:
@melissabarber This is the one new label introduced in this PR.

Reply:
I believe also this message is a bit of a fallback - we will soon deliver the story that hides the credit card field bundle when your config isn't available. Fairly sure.

Reply:
Yeah that story is here in the backlog: https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HtRpQAK/view
Do we want to not include this custom label and just not throw an error then for this WI?

Reply:
Sorry for chiming in late on this one. @kenlewis-sfdo if you were to throw an error that's not a custom label, what would that error look like? This is the copy I worked on for this error message, "You must have {0} enabled to use these fields." where {0} is the value for commonPaymentServices.

Reply:
I honestly like having this in here, even if it means a touch of cruft in the form of that label. Especially while we wait for that story to get in ahead of this, I like getting the feedback

Reply:
I removed it.  I agree that once that WI is done, this label won't ever be encountered.  It is a nice fallback, but maybe not necessary.  Now, with this exception gone, the pmt widget just spins forever when it is on a form and there is no config (the sdk isn't loaded).  I think this is acceptable for the interim until we can just block it from being added to a template when not yet configured.  But let me know if you feel strongly and I'll put it back!

Reply:
@melissabarber no worries at all about the timing!  Thank you for your collaboration and attention on this.  I appreciate it and I appreciate you!